### DISCUSSION TITLE
How to open a modal with nested screens

### DISCUSSION THREAD

**Discussion by Arjan-Zuidema** [react-navigation/react-navigation#11802] [Q&A]
I have a group with presentation modal:

```js
<Stack.Group
  screenOptions={{
    presentation: 'modal',
  }}
>
  <Stack.Screen
    name="test-modal"
    component={TestModalStack}
    options={{
      headerShown: false,
    }}
  />
</Stack.Group>
```

And TestModalStack looks like this:

```js
export default function TestModalStack() {
  return (
    <Stack.Navigator>
      <Stack.Screen name="test-start" component={TestModalStartScreen} />
      <Stack.Screen name="test-color" component={TestModalScreen} />
    </Stack.Navigator>
  );
}
```

I want to open the modal with a nested stack: `test-modal` -> `test-start` -> `test-color`
But I cannot seem to make it work. 

The code below just opens the modal on `test-start` and not the nested `test-color` screen.
```js
navigation.navigate('test-modal', {
  screen: 'test-start',
  params: {
    screen: 'test-color'
  }
})
```

And ideas how to open the modal on the `test-color` screen so it has a back button to go back to `test-start`?

**Comment by chriswetterman** [comment]
Interested in this as well. Any progress @Arjan-Zuidema?
